src/permit/Enforcement.cs

```diff
@@ -1,27 +1,30 @@
 ﻿using System;
+using System.Collections;
 using System.Collections.Generic;
 using System.Net;
 using System.Net.Http;
-using System.Threading.Tasks;
-using System.Text.Json;
-using PermitSDK.Models;
 using System.Net.Http.Headers;
 using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
-using System.Collections;
+using PermitSDK.Models;
+using PermitSDK.PDP.OpenAPI;
 
 public class PermissionCheckException : Exception
 {
     public PermissionCheckException() { }
 
-    public PermissionCheckException(string message) : base(message) { }
+    public PermissionCheckException(string message)
+        : base(message) { }
```

Comment:
Just making sure, this new line here is fine ?

Reply:
it complied and I have a test that uses this files (the .Net SDK tests are good, saved my **s few times :)